upup/models/cloudup/resources/addons/networking.cilium.io/k8s-1.16-v1.14.yaml.template

```
@@ -903,65 +1145,99 @@ spec:
       serviceAccount: cilium
       serviceAccountName: cilium
       terminationGracePeriodSeconds: 1
+      hostNetwork: true
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+          - labelSelector:
+              matchLabels:
+                k8s-app: cilium
+            topologyKey: kubernetes.io/hostname
+      nodeSelector:
+        kubernetes.io/os: linux
```

We want this set as node affinity.
```suggestion
      nodeSelector: null
```